pkgs/tools/misc/ifdtool/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "ifdtool-${version}";
+  version = "4.9";
+
+  src = fetchgit {
+    url = "http://review.coreboot.org/p/coreboot";
```

Comment:
the url passed here seems to be invalid, I get a 404 in the browser. Should be a git repository.

Also see
https://logs.nix.ci/?key=nixos/nixpkgs.55066&attempt_id=a6c7cad8-54c3-453b-ad07-b5548fc2c41c

Reply:
It might be that this `sha256sum`already existed in your nix-store - try replacing it in both expressions with known-wrong hashes (all zeroes) and build again.